server/src/actors/project.rs

```diff
@@ -792,7 +792,7 @@ pub struct ProjectCache {
     upstream: Addr<UpstreamRelay>,
     projects: HashMap<ProjectId, Addr<Project>>,
     local_states: HashMap<ProjectId, Arc<ProjectState>>,
-    state_channels: HashMap<ProjectId, oneshot::Sender<ProjectState>>,
+    state_channels: HashMap<ProjectId, (oneshot::Sender<ProjectState>, usize)>,
```

Comment:
Can we make this an `Instant` instead? Otherwise it might be hard to "guess" how long projects remain in here.

Reply:
done!